hathor/conf/testnet.yml

```diff
@@ -0,0 +1,37 @@
+P2PKH_VERSION_BYTE: x49
+MULTISIG_VERSION_BYTE: x87
+NETWORK_NAME: testnet-golf
+BOOTSTRAP_DNS:
+  - golf.testnet.hathor.network
+
+# Genesis stuff
+GENESIS_OUTPUT_SCRIPT: 76a914a584cf48b161e4a49223ed220df30037ab740e0088ac
+GENESIS_TIMESTAMP: 1577836800
+GENESIS_BLOCK_NONCE: 826272
+GENESIS_BLOCK_HASH: 0000033139d08176d1051fb3a272c3610457f0c7f686afbe0afe3d37f966db85
+GENESIS_TX1_NONCE: 190
+GENESIS_TX1_HASH: 00e161a6b0bee1781ea9300680913fb76fd0fac4acab527cd9626cc1514abdc9
+GENESIS_TX2_NONCE: 115
+GENESIS_TX2_HASH: 00975897028ceb037307327c953f5e7ad4d3f42402d71bd3d11ecb63ac39f01a
+
+# tx weight parameters. With these settings tx weight is always 8
+MIN_TX_WEIGHT_K: 0
+MIN_TX_WEIGHT_COEFFICIENT: 0
+MIN_TX_WEIGHT: 8
+CHECKPOINTS:
+  100_000: 0000007ece4c7830169f360ed11c51b776e1b72bf0060e6e5b325ca8be474ac5
+  200_000: 00000113ecd4b666116abf3d3f05ad509d903d6b456a1e8c35e46a9e426af11a
+  300_000: 000000e42df13e4e7490cee98f303cb3b0ca33f362af180c5f7df740c98699d9
+  400_000: 000000e9a748b34fc4d662d88bb36ef2a033ba129960924208be14eccdac1a65
+  500_000: 000000b5c4572d7b85e585849540ece44b73948c5cdbc6f17a9a3a77fbd0f29a
+  600_000: 000000f6743ba3d67e51d7adc21821b8263726ce3bc86010d5e1a905bf2531dc
+  700_000: 0000008fda01c9e5fd6f99a5461e6dbf1039cba38cc8d0fc738a097d71caa968
+  800_000: 000000397af32fcc4eeb6985d96326c1ff4644792631872a00394688b1782af5
+  900_000: 00000097ae405036614f4335ad0e631df8fc5f7434e82c3421627e2fea4e1830
+  1_000_000: 000000145ba662cdee0d72034658f93a0a3a4568d5ba5083ff09013ca1e6556c
+  1_100_000: 000000404e6ff6a23695a6ffe712ce1c4efc02e75bbc11c3129f4c2377b07743
+  1_200_000: 0000003be5fae5bb2c9ceaed589d172bcd9e74ca6c8d7d2ca06567f65cea7c9b
+  1_300_000: 0000000000007d39de6e781c377bc202213b0b5b60db14c13d0b16e06d6fd5ac
+  1_400_000: 000000000df9cb786c68a643a52a67c22ab54e8b8e41cbe9b761133f6c8abbfe
+  1_500_000: 000000000c3591805f4748480b59ac1788f754fc004930985a487580e2b5de8f
+  1_600_000: 00000000060adfdfd7d488d4d510b5779cf35a3c50df7bcff941fbb6957be4d2
```

Comment:
Just to confirm if my understanding is correct. On the YAML side the `_` character will make the key be a string (or maybe even keys are always strings), but because of pydantic (not even `_parse_checkpoints`, since it already expects a `Dict[int, str]`) the keys will somehow become `int`s. Can we double check that?

Reply:
Actually the `_` padding in numbers is part of the [YAML specification](https://yaml.org/type/int.html), and is not related to python (or at least, it shouldn't be).

The `pyyaml` lib we use for yaml parsing implements that, so it's not something done by pydantic. It works for numbers both in keys and values, in any part of the yaml file.

Reply:
Ooh, got it, thank you.